src/js/handlebars.helpers.js

```diff
@@ -102,6 +102,63 @@ export default function handlebarsHelpers(handlebars) {
     // Call the formatDate helper with the determined date and format
     return handlebars.helpers.formatDate(dateString, dateToFormat, format);
   });
+
+  /**
+   * Format time duration into a string.
```

Comment:
We really need to start getting unit testing on this helper file, yes, i'm a meanie raising now after its got so much already.

Reply:
Sounds like a separate new feature development.